Core/COEditingContext.m

```diff
@@ -911,26 +964,9 @@ a correct deletion status (critical to update the cross
 		}
 		else if (![deletedPersistentRootUUIDs containsObject: persistentRootUUID])
 		{
-			/* For a finalized persistent root, insertedOrUndeleted is nil */
-			COPersistentRoot *insertedOrUndeleted = [self persistentRootForUUID: persistentRootUUID];
-
-			if (insertedOrUndeleted != nil)
```

Comment:
If this piece of code is removed, how do you handle updating references to persistent root deleted done in another editing context?